services/app-api/forms/naaar.json

```diff
@@ -284,7 +284,7 @@
                 }
               ]
             },
-            "dashboardTitle": "Analysis method <br/> Frequency and plan utilization",
+            "dashboardTitle": "<div style=\"margin-left: 3rem;\">Analysis method <br/> Frequency and plan utilization</div>",
```

Comment:
I did try using the json structured styling that we have elsewhere ([for example](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/main/services/app-api/forms/mlr.json#L319-L324)) but it seemed to ignore the sx props for the dashboard title in the component

Reply:
nvm decided I didn't like this